pom.xml

```diff
@@ -10,12 +10,14 @@
   </parent>
 
   <artifactId>cloudbees-jenkins-advisor</artifactId>
-  <version>1.6-SNAPSHOT</version>
+  <version>2.10-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <properties>
-    <jenkins.version>2.19.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
+    <hpi-plugin.version>2.2</hpi-plugin.version>
```

Comment:
YAGNI, please update the Parent POM instead

Reply:
done